xml/art_raspberry-pi.xml

```diff
@@ -309,18 +218,19 @@
      </mediaobject>
     </figure>
     <para>
-     When started, &yast; shows an overview of available modules
-     (<xref linkend="fig-rpi-yast2"/>). Simply click an icon to open a module.
+     When started, &yast; shows an overview of available modules (<xref
+     linkend="fig-rpi-yast2"/>). To open the desired module, click on the
+     respective icon.
```

Comment:
Is the guide intended to also cover non-GUI operation? If so, we should mention the alternative: select the desired line and press the Return key.